client/src/js/components/bhStockExpired/bhStockExpired.js

```diff
@@ -0,0 +1,53 @@
+angular.module('bhima.components')
+  .component('bhStockExpired', {
+    templateUrl : 'js/components/bhStockExpired/bhStockExpired.html',
+    controller  : bhStockExpiredController,
+    transclude  : true,
+    bindings    : {
+      depotUuid : '<',
+      date : '<',
+    },
+  });
+
+bhStockExpiredController.$inject = ['StockService', 'moment', 'NotifyService'];
+
+/**
+ * service or depot selection component
+ */
+function bhStockExpiredController(Stock, moment, Notify) {
+  const $ctrl = this;
+  $ctrl.loading = false;
+  $ctrl.expiredInventories = [];
+
+  $ctrl.$onInit = () => {
+    expred();
+  };
+
+  $ctrl.$onChanges = () => {
+    expred();
+  };
+
+  /**
+   * @function expred
+   * get stock out inventories for a depot
+   */
+  function expred() {
```

Comment:
'exred()' ?  is this a typo?

Reply:
maybe because I setted ```is_expired``` to true instead of ```1```